docs/how-to/h-deploy-aws.md

```diff
@@ -0,0 +1,210 @@
+You can find more information about the available instance types in the [AWS documentation](https://us-east-1.console.aws.amazon.com/ec2/home?region=us-east-1#Instances:instanceState=running), 
+ensuring that you select the correct AWS.
+
+Deploy and integrate Kafka and ZooKeeper, for example:
+
+```shell
+juju deploy zookeeper -n3 --channel 3/stable
+juju deploy kafka -n3 --constraints "instance-type=m7i.xlarge" --channel 3/stable
+juju integrate kafka zookeeper
+```
+
+We also recommend to deploy a [Data Integrator](https://charmhub.io/data-integrator) for creating an admin user to manage the content of the Kafka cluster:
+
+```shell
+juju deploy data-integrator admin \
+  --config extra-user-roles=admin \
+  --config topic-name=admin-topic
+```
+
+And integrate it with the Kafka application:
+
+```shell
+juju integrate kafka admin
+```
+
+For more information on Data Integrator and how to use it, please refer to the [how-to manage applications](/t/charmed-kafka-how-to-manage-app/10285) guide.
+
+## Clean up
+
+[note type="caution"]
+Always clean AWS resources that are no longer necessary! Abandoned resources are tricky to detect and they can become expensive over time.
+[/note]
+
+To list all controllers use the `juju controllers` command.
+
+To destroy the Juju controller and remove AWS instance (**Warning**: all your data will be permanently deleted):
+
+```shell
+juju destroy-controller <CONTROLLER_NAME> --destroy-all-models --destroy-storage --force
+```
+
+Use `juju list-controllers` to list the available controllers the client has registered to. 
+
+After destroying the controller, check and manually delete all unnecessary AWS EC2 instances, to show the list of all your EC2 instances run the following command (make sure to use the correct region):
+ 
+```shell
+aws ec2 describe-instances --region us-east-1 --query "Reservations[].Instances[*].{InstanceType: InstanceType, InstanceId: InstanceId, State: State.Name}" --output table
+```
+
+[details="Output example"]
+```shell
+-------------------------------------------------------
+|                  DescribeInstances                  |
++---------------------+----------------+--------------+
+|     InstanceId      | InstanceType   |    State     |
++---------------------+----------------+--------------+
+|  i-0f374435695ffc54c|  m7i.xlarge    |  terminated  |
+|  i-0e1e8279f6b2a08e0|  m7i.xlarge    |  terminated  |
+|  i-061e0d10d36c8cffe|  m7i.xlarge    |  terminated  |
+|  i-0f4615983d113166d|  m7i.xlarge    |  terminated  |
++---------------------+----------------+--------------+
+```
+[/details]
+
+List your Juju credentials with the `juju credentials` command:
+
+```shell
+...
+Client Credentials:
+Cloud        Credentials
+aws          NAME_OF_YOUR_CREDENTIAL
+...
+```
+Remove AWS EC2 CLI credentials from Juju:
+
+```shell
+juju remove-credential aws NAME_OF_YOUR_CREDENTIAL
+```
+
+Finally, remove AWS CLI user credentials (to avoid forgetting and leaking):
```

Comment:
alternativevly, you could show the user how to delete the credentials or rotate them in the AWS web console.

Reply:
I have refactored this part. Please re-review